website/docs/d/rds_orderable_db_instance.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "RDS"
+layout: "aws"
+page_title: "AWS: aws_rds_orderable_db_instance"
+description: |-
+  Information about RDS orderable DB instances.
+---
+
+# Data Source: aws_rds_orderable_db_instance
+
+Information about RDS orderable DB instances.
+
+## Example Usage
+
+```hcl
+data "aws_rds_orderable_db_instance" "test" {
+  db_instance_class = "db.t2.small"
+  engine            = "mysql"
+  engine_version    = "5.7.22"
+  license_model     = "general-public-license"
+  storage_type      = "standard"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `engine` - (Required) DB engine. Engine values include `aurora`, `aurora-mysql`, `aurora-postgresql`, `docdb`, `mariadb`, `mysql`, `neptune`, `oracle-ee`, `oracle-se`, `oracle-se1`, `oracle-se2`, `postgres`, `sqlserver-ee`, `sqlserver-ex`, `sqlserver-se`, and `sqlserver-web`.
+* `availability_zone_group` - (Optional) Availability zone group.
+* `db_instance_class` - (Optional) DB instance class. Examples of classes are `db.m3.2xlarge`, `db.t2.small`, and `db.m3.medium`.
+* `engine_version` - (Optional) Version of the DB engine.
+* `engine_version_prefix` - (Optional) Prefix of the version of the DB engine. NOTE: This attempts to return only the latest version with the given prefix but may not understand version numbers for all engines.
```

Comment:
While I totally understand the nice utility this would provide, this functionality seems farther away from our normal "we only support want to support what the API supports" design principle in the provider. In particular, I have some concerns about the upkeep of this functionality because RDS has very inconsistent engine version naming across engines that is not necessarily just numbers or alphabetical. I think this would be much better suited as a feature request to AWS itself, rather than something we try to support client-side in the provider.

Let's remove this for now. 👍

Reply:
Done.